SmartDeviceLink/SDLError.m

```diff
@@ -228,6 +228,14 @@ + (NSError *)sdl_choiceSetManager_choiceUploadFailed:(NSDictionary *)userInfo {
     return [NSError errorWithDomain:SDLErrorDomainChoiceSetManager code:SDLChoiceSetManagerErrorUploadFailed userInfo:userInfo];
 }
 
++ (NSError *)sdl_choiceSetManager_failedToCreateMenuItems {
+    NSDictionary<NSString *, NSString *> *userInfo = @{
+                                                       NSLocalizedDescriptionKey: NSLocalizedString(@"Choice Set Manager error", nil),
+                                                       NSLocalizedFailureReasonErrorKey: NSLocalizedString(@"Choice set manager failed to create menu items do to menuName being empty.", nil),
+                                                       };
```

Comment:
Add `NSLocalizedRecoverySuggestionErrorKey: NSLocalizedString(@"If you are setting the `menuName`, it is possible that the head unit is sending incorrect `displayCapabilities`.")`

Reply:
fixed!